src/main/java/com/stripe/model/SubscriptionScheduleRevision.java

```diff
@@ -0,0 +1,121 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiResource;
+import java.util.List;
+import java.util.Map;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class SubscriptionScheduleRevision extends StripeObject implements HasId {
+  /** Time at which the object was created. Measured in seconds since the Unix epoch. */
```

Comment:
Should all of those be real javadocs? (sorry train of thoughts, nothing specific to that property, just wondered)